airflow/models/abstractoperator.py

```diff
@@ -181,7 +181,9 @@ def resolve_template_files(self) -> None:
                             try:
                                 content[i] = env.loader.get_source(env, item)[0]  # type: ignore
                             except Exception as e:
-                                self.log.exception(e)
+                                self.log.exception(
+                                    "Failed to get source with %s due to %s", item, str(e)
+                                )
```

Comment:
```suggestion
                                self.log.exception("Failed to get source %s", item)
```

`exception` already prints the exception, don’t need to repeat it.

Reply:
nice, thanks. updated